### PR TITLE
Milestone 3 : Add Contact Form

### PR DESCRIPTION
**Pull request by AngomRanjan**
## Profile Milestone 3 : Add Contact Form

In this milestone : 

- Added Footer Section
- Added A Contact Form
- Added HTML5 Form Validation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -221,6 +221,22 @@ <h3 class="head3" id="social-2">LET’S CONNECT</h3>
...
     </main>
+    <footer class="footer" id="contacts">
+      <div class="content-placeholder" id="contact-placeholder">
+        <h2 class="head-1" id="contact-me">Contact Me</h2>
+        <p id="contact-msg">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>    
+        <form class="form" action="https://formspree.io/f/xbjwpzkw" method="post">
```
Comment (gtekle) [inline index.html]: - [ ] Kindly use Flexbox to layout the elements as per the project requirement. 
contact-form-elem-flex-layout

File: index.html [hunk 2]
```diff
@@ -221,6 +221,22 @@ <h3 class="head3" id="social-2">LET’S CONNECT</h3>
...
+          <input class="form-input input-line" type="text" placeholder="Enter your name" name="name" maxlength="30" required/>
+          <input class="form-input input-line" type="email" id="email" name="email" placeholder="yourid@gmail.com" required/>
+          <textarea class="form-input" rows="10" maxlength="500" placeholder="Write your message here" name="message" required ></textarea>
+          <input class="btn" type="submit" value="Get It Touch" />
```
Comment (gtekle) [inline index.html:236]: - [ ] Kindly align left the button as in the Figma design for mobile version. On the desktop version you will have it at the center, but is in your next milestone.
### Design
contact-form-button-left-figma
### Impl
contact-form-button-centered-impl

- [ ] Kindly use the background color during `active` state (while the button is pressed) is `#2230D2`.
```
btn:active {
}
```
pressed-button-bg


- [ Optional ] It is good if you can pointer cursor when you hover on the button for a better user experience by indicating it is clickable. `cursor: pointer;`

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @AngomRanjan :wave: , 

### Splendid! You have done an amazing job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
this-close


## Highlights
:heavy_check_mark: Clean and well organized code
:heavy_check_mark: Stunning UI design
:heavy_check_mark: Perfect use of Github flow
:heavy_check_mark: Descriptive PR title and summary
:heavy_check_mark: Meaningful commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ Optional ] Kindly use the background color as in the Figma design for the body.
   #### Design
mobile-body-bg-color-figma

   #### Impl
mobile-body-bg-color-impl


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AngomRanjan** [comment]
Made The Changes

**Review by keroloussamy — APPROVED** [review]
Hi @AngomRanjan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gtekle** [comment]
> Made The Changes

:+1:

**Pull request merged**